tests/testthat/test-exporters.R

```diff
@@ -183,3 +181,35 @@ test_that("export_as_pdf works", {
 
   expect_equal(res$npages, 2)
 })
+
+test_that("exporting lists of tables and listings works", {
+  bmf <- basic_matrix_form(mtcars)
+  blmf <- basic_listing_mf(mtcars)
+  l_mf <- list(bmf, blmf)
+
+  output <- export_as_txt(l_mf, page_num = "page {i} of {n}", cpp = 90)
+  last_line_last_page <- strsplit(output, "\n")[[1]][168]
+
+  expect_true(grepl(last_line_last_page, pattern = "page 4 of 4"))
+  expect_equal(nchar(last_line_last_page), 90)
+
+  suppressWarnings(expect_warning(
+    export_as_txt(l_mf, paginate = FALSE),
+    "paginate is FALSE, but x is a list of tables or listings, so paginate will be set to TRUE"
+  ))
+
+  # export_as_pdf
+  tmpf <- tempfile(fileext = ".pdf")
+  tmpf <- "to_delete.pdf"
+  output <- export_as_pdf(l_mf, file = tmpf, page_num = "page {i} of {n}", cpp = 90)
```

Comment:
This fails with 
```r
Error in eval(cur_call) : object 'cur_gpar' not found
```
@edelarua do you have any idea why?

Reply:
My guess is that it is preparing for the wrong number of pages

Reply:
I'm not sure why - it seems like some info is being lost somewhere along the way. The error occurs in `font_lcpi`.